gibbon-compiler/src/Gibbon/HaskellFrontend.hs

```diff
@@ -442,6 +442,7 @@ primMap = M.fromList
   , ("sqrt", FSqrtP)
   , ("==", EqIntP)
   , (".==.", EqFloatP)
+  , ("*==*", EqCharP)
```

Comment:
Looks good!

Eventually we should have just one `EqLit` primitive that can compare all of these literals...

Reply:
Makes sense, maybe we make an issue about this so its not forgotten in the future?